### ISSUE TITLE
YouTube tabs get erroneously discarded with a not-yet-submitted comment on the tab.

### ISSUE THREAD

**Issue by EricTheDad** [rNeomy/auto-tab-discard#339]
I have the `Do not discard a tab when form changes are not yet submitted` option set, but I frequently, albeit not consistently, have tabs where I'm working on a YouTube comment get discarded, losing my entire comment in the process, when I open a new tab to look up a definition or double check a fact related to the comment I'm writing.

Possible key circumstances:
- I tend to have a lot of YouTube tabs open at the same time. My primary reason for using Auto Tab Discard is so I can use tabs as temporary bookmarks for videos I want to watch in the near future.
- I don't think the erroneous tab discard has ever happened on the first time I've opened another tab along side a tab with an unsubmitted comment.  It takes a few cycles of opening a new tab, using it, then closing it to return to the comment before the tab with the comment is erroneously discarded.

I'll update this comment if I manage to find consistent repro steps.

**Comment by ErikAnderson3** [comment]
I've seen the same behavior for some time now on Reddit and other sites.  This drives me nuts.  I participate in r/etymology, among others, where looking things up is a frequent occurrence.  If I forget to copy-paste my comment text into Notepad++, I will often lose my comment entirely when Auto Tab Discard erroneously discards the tab while I'm busy on other tabs double-checking term derivations and the like.  **Exceedingly frustrating.**

I've noticed that this happens when I open multiple other tabs before coming back to the one I'm entering text into.  Not sure yet about any thresholds with regard to how many tabs, or how long time-wise, before returning to the text-entry tab.

In short, `Do not discard a tab when form changes are not yet submitted` doesn't seem to work.  And it's not just YouTube.
